.editorconfig

```diff
@@ -0,0 +1,29 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*.rs]
```

Comment:
I don't think these are necessary as `rustfmt` automatically takes care of rust formatting.